app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,126 @@
+      direction LR
+      KongGatewayService[Kong Gateway service 1]
+      PagerDutyServiceMapping[PagerDuty service 1]
+      GitHubRepoMapping[GitHub repository 1]
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService[Kong Gateway service 2]
+      AnotherService[PagerDuty service 2]
+      DifferentRepo[GitHub repository 2]
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  GitHub -- "GitHub repositories" --> KonnectServiceCatalog
+  PagerDuty -- "PagerDuty services" --> KonnectServiceCatalog
+  KongGateway -- "Kong Gateway services" --> KonnectServiceCatalog
+
+  %% Style the subgraphs
+  class KonnectServiceCatalog konnect
+  class BillingService serviceCatalog
+  class InventoryService serviceCatalog
+{% endmermaid %}
+<!-- vale on-->
+
+> Figure 1: This diagram shows how you can use both external integrations, like GitHub and PagerDuty, as well as built-in integrations like {{site.base_gateway}} to pull resources into Service Catalog. You can then map those resources (like GitHub repositories, PagerDuty services, and {{site.base_gateway}} services) to Service Catalog Services.
+
+## Service Catalog use cases
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Keep track of your organization's service ownership by mapping teams to services. | [{{site.konnect_short_name}} Organization Teams](https://cloud.konghq.comorganization/teams) |
+| Gain visibility into all your services, including unrecognized or undiscovered APIs in your organization.  | [Service Catalog integrations](https://cloud.konghq.com/servicehub/integrations) |
+| Consolidate key service health metrics into a single dashboard, allowing you to interact with other tools from one place. | [Service Catalog dashboard](https://cloud.konghq.com/servicehub) |
+| Govern how services are created and maintained across your company to adhere to security, compliance, and engineering best practices. | ? |
+
+## Service Catalog terminology
+
+| Term | Definition |
+| ---- | ---------- |
+| Integration | These are applications, either internal {{site.konnect_short_name}} applications or external applications, from which you can ingest data. For example, GitHub. |
+| Resource | Resources are entities that are injested from integrations. For example, a GitHub repository. |
+| Service | A Service is a collection of resources in the Service Catalog. It represents what you might consider a “service”, typically defined as an independent system delivering specific capabilities and owned by a singular team in your organization. It is *not* the same as a {{site.base_gateway}} [service](/gateway/latest/key-concepts/services/). For example, a Service Catalog Service could have 10 {{site.base_gateway}} services, a GitHub repository, and several PagerDuty services mapped to it. |
```

Comment:
1) Should we capitalize the "R" in "Resource" (and "I" in "Integration, for that matter) whenever we refer to the actual entities? Not sure whether we do this for other Docs pages.
2) I worry that the last sentence might be confusing, since we would need to clarify that both the "Gateway Service" and the "PagerDuty Service" are actually Resources, even though they contain the word 'service' in their names.